python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -73,9 +74,10 @@ def _get_basic_autoscaling_config() -> dict:
                     "Custom3": 1,
                 },
             },
-            # Same as "small-group" with a GPU entry added.
+            # Same as "small-group" with a GPU resource entry added
+            # and modified max_workers.
             "gpu-group": {
-                "max_workers": 300,
+                "max_workers": 200,
```

Comment:
I am not familiar with autoscaler. What is the relationship between "max_workers" in autoscaler and "maxReplicas" in RayCluster CR YAML file? Thank you!

Reply:
maxReplicas is the KubeRay terminology and max_workers is the equivalent autoscaler terminology.
It's just formatting / config translation.

Reply:
Thanks!